### ISSUE TITLE
Remove metadata md files in favor of redirecting to schema documenation

### ISSUE THREAD

**Issue by aclum** [microbiomedata/NMDC_documentation#59]
I propose deleting everything in the folder https://github.com/microbiomedata/NMDC_documentation/tree/main/docs/reference/metadata in favor of redirecting to the schema autogenerated documentation (https://microbiomedata.github.io/nmdc-schema/)? Direct links to these markdown files in reads the docs don't exist any more but the links still work and the documentation is not being kept up to date.  

Before we do this we'd have to remove references to these in https://github.com/microbiomedata/NMDC_documentation/blob/main/docs/howto_guides/api_gui.md
 and identify if there are any other references to this folder.

cc @turbomam @ljohnson09

**Comment by mslarae13** [comment]
I agree with this, and propose archiving this repo if we can in favor of
https://github.com/microbiomedata/docs/tree/main/content/home/src/

**Comment by ssarrafan** [comment]
Backlog label, removing from sprint board.
